app/grants/views.py

```diff
@@ -1053,18 +1091,18 @@ def grants_by_grant_clr(request, clr_round):
     if len(pks):
         increment_view_count.delay(pks, grants[0].content_type, request.user.id, 'index')
 
-    current_partners = MatchPledge.objects.filter(clr_round_num=clr_round)
-    current_partners_fund = 0
+    # current_partners = MatchPledge.objects.filter(clr_round_num=clr_round)
+    # current_partners_fund = 0
 
-    for partner in current_partners:
-        current_partners_fund += partner.amount
+    # for partner in current_partners:
+    #     current_partners_fund += partner.amount
 
     grant_types = get_grant_clr_types(clr_round, network=network)
 
-    grants_following = Favorite.objects.none()
+    # grants_following = Favorite.objects.none()
     collections = []
     if request.user.is_authenticated:
-        grants_following = Favorite.objects.filter(user=request.user, activity=None).count()
+        # grants_following = Favorite.objects.filter(user=request.user, activity=None).count()
```

Comment:
Following/Favourite is still an option in the "me menu" - do we want to be removing this?